### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_Reflow: Data is getting lost at 400% zoom

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14398]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.10;#AINotInScope;#Zoom;

### Prerequisite: 

- Set resolution to 1280*1024
- Set scaling to 400

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Check if data is getting lost at 400% zoom

### Actual:
Data is getting lost at 400% zoom. The radio buttons "SQL Database" and "SQL Edge" are not properly visible also the edit fields are not properly visible.

Same issue is also observed throughout the application

### Expected:
All the data should be visible at 400% zoom

### User Impact:
Low Vision users will face difficulty in getting the data.

**Comment by hannahqin** [comment]
We have filed a Reflow exception request and are waiting on official approval from CELA. We received initial support from Guy Barker. I'll keep this thread updated once we hear back on the official decision.
https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by alanrenmsft** [comment]
closing the issue, Azure Data Studio requested exception for WCAG 2.1: https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by fsteffi** [comment]
![image](https://user-images.githubusercontent.com/24580570/221148013-c036177a-1bbe-4bbc-a022-96bbedabca27.png)

visible with new method
Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300
GithubTags:#Closed;
